fuzz.sh

```diff
@@ -5,6 +5,8 @@ command -v afl-fuzz >/dev/null 2>&1 || { echo >&2 "Please install afl-fuzz."; ex
 
 mkdir build.fuzz
 cd build.fuzz
-CC=afl-clang cmake -G Ninja .. -DAVIF_CODEC_AOM=0 -DAVIF_BUILD_AOM=0 -DAVIF_CODEC_DAV1D=1 -DAVIF_LOCAL_DAV1D=1 -DAVIF_BUILD_TESTS=1 || exit 1
+# Enable the "-DAVIF_BUILD_EXAMPLES=1" to get the target fuzz test program.
+#  e.g: avif_example_decode_file or avif_example_decode_streaming
+CC=afl-clang cmake -G Ninja .. -DAVIF_BUILD_EXAMPLES=1 -DAVIF_CODEC_AOM=0 -DAVIF_BUILD_AOM=0 -DAVIF_CODEC_DAV1D=1 -DAVIF_LOCAL_DAV1D=1 -DAVIF_BUILD_TESTS=1 || exit 1
```

Comment:
Joe: Could you take a look at this pull request? Thanks.

@sdt27 I am very sorry for the late review. I was out of the office.

I am not familiar with fuzz.sh. It seems that fuzz.sh is out of date. For example, the last line of fuzz.sh refers to `../tests/inputs`, but that directory is not in the libavif source tree. Also, I can't find the `aviffuzz` test program mentioned in the last line.

Reply:
We should probably remove `fuzz.sh`, as I believe it is obsoleted by our engagement with `oss-fuzz`. This script is attempting to invoke things that don't exist anymore. We should not accept this PR.

Reply:
Joe: Thank you for your reply. I will decline this PR and write a new PR to remove fuzz.sh.